### ISSUE TITLE
Option to "replace all" or "install as variant" upon collection install

### ISSUE THREAD

**Issue by otaviotscha** [Nexus-Mods/Vortex#14654]
I just started using Vortex collection feature, but I am suffering from the following problem: I was installing [this collection](https://next.nexusmods.com/skyrimspecialedition/collections/rvxts8), then it started to show I/O errors thousands of times along the way. So I nuked (wiped everything) from my Skyrim folder as suggested by the curator on the Discord Server and started fresh, but, now I am prompted all the time asking me if I want to "replace"or "install as variant" as follows:

![image](https://github.com/Nexus-Mods/Vortex/assets/15371120/608fc4d4-dbdc-45fa-90ce-ad090f0f6011)

Please, add a configuration or a single prompt when you start the collection install to either automatically respond with "replace all" or "install as variant" for all cases.

**Comment by captainshrimps** [comment]
Yes, I really hope this feature gets added. It's so annoying and makes premium feel like not premium.

**Comment by jonatino** [comment]
This would be an amazing QoL addition.

**Comment by mrrodgerz1-ctrl** [comment]
Encountering this same thing when installing multiple collections, please reopen this
